docs/configuration.md

```diff
@@ -81,9 +81,11 @@ use Lcobucci\JWT\Signer;
 use Lcobucci\JWT\Signer\Key\LocalFileReference;
 use Lcobucci\JWT\Signer\Key\InMemory;
 
+// You may use RSA OR ECDSA and all their variations (256, 384, and 512) and EdDSA over Curve25519
+$signer        = new Signer\Rsa\Sha512();
+$signer        = Signer\Ecdsa\Sha512::create();
```

Comment:
I think we should use an alert to convey this difference (instead of re-assigning the variable).